docs/usage.adoc

```diff
@@ -46,6 +46,28 @@ If you only want to use smart testing on `surefire` or `failsafe` plugin definit
 
 To do it you just need to set `smart.testing.apply.to` with either `surefire` or `failsafe` to just enable in one of them.
 
+=== Skipping Tests
+
+To skip running the tests for a particular project, Smart Testing extension provides support with maven's skip test 
+functionality offered by system properties `skipTests`, `skipITs` and `maven.test.skip`.
+
+Set the `skipTests` property to true, to skip running the tests.
+
+`$ mvn install -DskipTests`
+
+Since `skipTests` is supported by both Surefire and Failsafe Plugin, this will have the effect of not running any tests.
+If, instead, you want to skip only the integration tests being run by the Failsafe Plugin, you would use the `skipITs` 
+property.
+
+`$ mvn install -DskipITs`
+
+To skip execution as well as compiling the tests, you can use the `maven.test.skip` property.
+`maven.test.skip` is honored by both Surefire and Failsafe Plugins.
+
+`$ mvn install -Dmaven.test.skip=true`
+
+NOTE: Setting `skipTests` or `maven.test.skip` also disables smart testing extension.
+
```

Comment:
I think that this describes the behavior of surefire, which can be easily traceable: http://maven.apache.org/surefire/maven-surefire-plugin/examples/skipping-tests.html and similar... 
What I would expect from this text is describing what happens (from the Smart Testing point of view) when somebody uses the properties...

Reply:
Well, this information is briefly contained in line 69. The rest of the text describes only the "Surefire skip tests" behavior - this I can read in the Surefire documentation. What I want to know is what happens (with Smart Testing tool) if I use any of the properties: `skipTests`, `skipITs` or `maven.test.skip` and why.